docs/sources/setup-grafana/configure-security/configure-authentication/okta/index.md

```diff
@@ -84,7 +84,7 @@ The following table outlines the various Okta OAuth2 configuration options. You
 | `skip_org_role_sync`    | No       | Set to `true` to stop automatically syncing user roles. This will allow you to set organization roles for your users from within Grafana manually.                                                                                                                                                                                                                                                                                                                                                                             | `false`                       |
 | `allowed_groups`        | No       | List of comma- or space-separated groups. The user should be a member of at least one group to log in.                                                                                                                                                                                                                                                                                                                                                                                                                         |                               |
 | `allowed_domains`       | No       | List comma- or space-separated domains. The user should belong to at least one domain to log in.                                                                                                                                                                                                                                                                                                                                                                                                                               |                               |
-| `use_pkce`              | No       | Set to `true` to use [Proof Key for Code Exchange (PKCE)](https://datatracker.ietf.org/doc/html/rfc7636). Grafana uses the SHA256 based `S256` challenge method and a 128 bytes (base64url encoded) code verifier.                                                                                                                                                                                                                                                                                                             | `false`                       |
+| `use_pkce`              | No       | Set to `true` to use [Proof Key for Code Exchange (PKCE)](https://datatracker.ietf.org/doc/html/rfc7636). Grafana uses the SHA256 based `S256` challenge method and a 128 bytes (base64url encoded) code verifier.                                                                                                                                                                                                                                                                                                             | `true`                        |
```

Comment:
@arukiidou , I'm not sure this change is accurate. I believe the `false` value in the docs represents the default value, but that they should set it to `true` 

@linoman , can you please review this PR and let me know your thoughts? Thank you!

Reply:
@chri2547 
Thank you again.

Yes. I think this change is correct.

```yaml
services:
  grafana:
    image: grafana/grafana:10.1.0
    restart: always
    ports:
      - "3000:3000"
    environment:
      GF_SECURITY_ADMIN_USER: "grafana"
      GF_SECURITY_ADMIN_PASSWORD: "grafana"
```

![image](https://github.com/grafana/grafana/assets/10738333/764ca989-8ce4-4d08-96d9-503f28970ee1)

Reply:
This change is correct. The default value has been set to `true`.